app/src/main/kotlin/com/intive/picover/common/viewmodel/state/MVIStateType.kt

```diff
@@ -0,0 +1,7 @@
+package com.intive.picover.common.viewmodel.state
+
+enum class MVIStateType {
+	LOADING,
+	ERROR,
+	LOADED,
+}
```

Comment:
I would vote for nesting `MVIStateType` in `MVIState` since both are small, tiny classes. The result would be `MVIState.Type`

Reply:
Done